src/frontend/src/common/components/library/data_table/components/HeaderRow/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { isEqual } from "lodash";
```

Comment:
These file names are a little confusing, it's not immediately clear to me how `HeaderRow` is different from `TableHeader`.

Reply:
hmm i see so HeaderRow is an implementation of TableHeader, still a little confusing but not as confusing as i originally thought, no change needed